packages/cache-redis/src/initialize/index.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  ExternalModuleDeclaration,
+  InternalModuleDeclaration,
+  MedusaModule,
+} from "@medusajs/modules-sdk"
+import { ICacheService } from "@medusajs/types"
+import { RedisCacheModuleOptions } from "../types"
+
+export const initialize = async (
+  options?: RedisCacheModuleOptions | ExternalModuleDeclaration
+): Promise<ICacheService> => {
+  const serviceKey = "cacheService"
```

Comment:
suggestion: would it make sense to import the key from the module definition exported by the module-sdk? The module SDK could export a map along side the definition so that there can't be discrepancies between those modules and the definition key if there is a typo of something. wdyt? if you agree, we can do that in all the modules.

Reply:
Think this is an excellent idea :)